lib/matplotlib/_color_data.py

```diff
@@ -2,14 +2,15 @@
 
 
 BASE_COLORS = {
-    'b': (0, 0, 1),
-    'g': (0, 0.5, 0),
-    'r': (1, 0, 0),
-    'c': (0, 0.75, 0.75),
-    'm': (0.75, 0, 0.75),
-    'y': (0.75, 0.75, 0),
-    'k': (0, 0, 0),
-    'w': (1, 1, 1)}
+    'b': (0, 0, 1),        # Blue
```

Comment:
I think the capitalization to indicate the representing character is a bit too sophisticated. It's the first character except for black, where 'k' and 'K' are also quite similar. At first I was wondering why you capitalize all colors. I'd stick always with lowercase, also in the cases below.

Reply:
I see. Thanks for reviewing!